tensorflow/java/src/gen/cc/ops/op_generator.cc

```diff
@@ -0,0 +1,52 @@
+/* Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Oh, also, I think we can simplify the directory structure, don't need all these subdirectories.
How about placing all these files in `tensorflow/javasrc/gen`? (So, `tensorflow/java/src/gen/op_generator{.h,.cc,_main.cc}` etc. instead of the `cc/ops` subdirectories)